README.md

```diff
@@ -36,9 +36,13 @@ The value of `sonar.web.context`. Setting this to something like `/sonar` allows
 
     sonar_plugins_dir: /usr/local/sonar/extensions/plugins
     sonar_plugins_url: []
+    plugins_to_remove: []
```

Comment:
How about remove plugins that are not listed in 'sonar_plugins_url' instead of specifying what to remove.

Reply:
There are some plugins that installed by default when you install SQ and it's sometimes changing in new releases of SQ, you don't want to remove those plugins.
And you need this variable also to upgrade some plugins and remove the old one.

Reply:
I'm ok with removing default plugins :) - the idea is that everything is under control of the maintainer and remove any ambiguity. You can create this variable from what was found on the system and what should be from configuration management.